Orchestrator.py

```diff
@@ -46,14 +46,15 @@ def __init__(self, numRobots, initX, initY):
                    'y':                initY,
                    'heading':          0,
                    'speed':            0,
-                   'movementDuration': 0
                 }
             ) for i in range(1, self.numRobots+1)
         ])
 
         # initial movements
         for dotBotId in range(1,self.numRobots+1):
-            self._updateMovement(source=dotBotId)
+            heading, speed = self._pickNewMovement(dotBotId)
```

Comment:
use format `(heading, speed)`. In Python using `heading, speed =` is valid and you implicitly assume it's a tuple, but it;s always better to be super explicit